lib/cocoapods/installer/analyzer/analysis_result.rb

```diff
@@ -76,6 +81,19 @@ def sandbox_needs_install?
           needing_install = state.added.length + state.changed.length + state.deleted.length
           needing_install > 0
         end
+
+        private
+
+        # @return [Array<PodTarget>] The model representations of pod targets
+        #         generated as result of the analyzer.
+        #
+        def calculate_pod_targets(aggregate_targets)
```

Comment:
is this needed?

Reply:
seems like it's doing the same work done before passing the targets to the analyzer's initializer